3-fleetscope/modules/install_kueue/main.tf

```diff
@@ -0,0 +1,78 @@
+/**
```

Comment:
Just a thought - Is this really an `install_kueue` module, or already pretty close to a "generic" retrieve manifest, update to use AR remote, and apply to cluster module?

The later in-theory could be used for other apps in the future.

Reply:
@caetano-colin - We can do this as a follow-up.